ocs_ci/ocs/elasticsearch.py

```diff
@@ -1,23 +1,73 @@
 """
-Deploying en Elasticsearch server for collecting logs from ripsaw benchmarks
+Deploying en Elasticsearch server for collecting logs from ripsaw benchmarks.
+Interface for the Performance ElasticSearch server
 
 """
-import os
 import logging
 import base64
-import signal
-import subprocess
 import time
+import json
 
 from elasticsearch import Elasticsearch, exceptions as esexp
 
 from ocs_ci.ocs import constants
 from ocs_ci.ocs.ocp import OCP
 from ocs_ci.ocs.utils import get_pod_name_by_pattern
 from ocs_ci.utility.utils import TimeoutSampler
+from ocs_ci.helpers.performance_lib import run_command
 
 log = logging.getLogger(__name__)
 
+# mute the elasticsearch logging
+es_log = logging.getLogger("elasticsearch")
+es_log.setLevel(logging.CRITICAL)
+
+
+def elasticsearch_load(connection, target_path):
+    """
+    Load all data from target_path/results into an elasticsearch (es) server.
+
+    Args:
+        connection (dict): a dictionary with the main elasticsearch server information
+          {host: the server ip address, port: the port to connect}
+        target_path (str): the path where data was dumped into
+
+    Returns:
+        bool: True if loading data succeed, False otherwise
+
+    """
+
+    all_files = run_command(f"ls {target_path}/results/", out_format="list")
+    if "Error in command" in all_files:
+        log.error("There is No data to load into ES server")
+        return False
+    else:
+        log.info(f"Loading data from {target_path} to ES server at {connection}")
+        try:
+            log.info(f"Creating new ES connection to {connection}")
+            main_es = Elasticsearch([connection])
+            log.debug(f"Connection info : {main_es}")
+        except esexp.ConnectionError:
+            log.error(f"Can not connect to the main ES server on {connection}")
+            return False
+        for ind in all_files:
+            if ".data." in ind:  # load only data files and noy mapping info
```

Comment:
please fix comment from # load only data files and noy mapping info 
to
# load only data files and not mapping info

Reply:
done